synfig-studio/src/gui/widgets/widget_waypoint.h

```diff
@@ -57,15 +58,18 @@ class Widget_Waypoint;
 
 class Widget_Waypoint : public Gtk::Alignment
 {
-    Widget_ValueBase *value_widget;
+	Widget_ValueBase *value_widget;
 	Gtk::Label *value_node_label;
 	Gtk::Label *label;
 	Widget_Time *time_widget;
 	mutable synfig::Waypoint waypoint;
 	synfig::Canvas::Handle canvas;
 
-	Widget_Enum *before_options,*after_options;
+	Widget_Enum *before_options, *after_options;
 
+	// TCB Parameter members
+	Gtk::Frame *tcbFrame;
+	Gtk::Label *tensionLabel, *continuityLabel, *biasLabel, *temporalTensionLabel;
```

Comment:
I think this isn't necessary anymore :)

Reply:
Strange...
Those declaration members are not even initialized on the implementation file.

Constructors are for initializing members. I suspect the compilation might fail if I remove those members. Let me try.

Reply:
Yup, compilation failed.

And I moved those variables to the header file in the first commit :) Now I feel stupid I might even forget I made this PR two days from now XD

New line 182, 196, 203, 210, and 217
https://github.com/synfig/synfig/pull/1182/commits/220e76695b20fe63edfa42944b10e5a7ad33612b

Just out of curiosity, I removed those labels and redeclared in the constructor to see if compilation will fail. And it did. Because they are local scoped within the constructor.

Reply:
Sorry. I selected line 
Gtk::Frame *tcbFrame;
By mistake.

Reply:
OK, got what you meant. Move those labels back to the implementation file.
Committing the changes right away :)